### ISSUE TITLE
Links with images say only the “link” when using Firefox Version 57 Beta on NVDA screen reader

### ISSUE THREAD

**Issue by kimidesu** [noizu/public-bladeofeternity.com#176]
Reporter:	BlindDude [21737]
Submission Date:	November 22, 2017 08:37:10

Links with a pic say link -- Hi since the firefox 57 update all the links with picture labels only say "link" instead of news  or whatever it's supposed to say. Mostly limited to control panel but does the same thing for links at the bottom  that use image labels. The labeled links work right using IE 11 (edge doesn't work for blind users). I'm using windows 10 pro fall creators update,nvda as my screen reader & firefox 58 beta. Just to say it was messed up on 57 beta but none of the earlier beta builds. Thanks for the great non-sighted (https://bladeofeternity.com/bugs/detail/id/782)
